### ISSUE TITLE
Works with All Tesla Models?

### ISSUE THREAD

**Issue by MichaelH2023** [Kemmey/Kemmey-TeslaWatch-Public#533]
Does the TeslaWatch app work with all of the Tesla models. I am specifically interested in the Model X 2020. It would be great if this was noted in the app listing on the App Store.

**Comment by Kemmey** [comment]
Yeah - works with models S, X, Y and 3. Bluetooth direct connectivity requires phone key support - so only 3, Y and 2021 refreshed S and X.

**Comment by MichaelH2023** [comment]
Awesome! I'm buying the app now... Thank You!

**Comment by MichaelH2023** [comment]
Oops... missed this important part. Bluetooth direct connectivity requires phone key support - so only 3, Y and 2021 refreshed S and X. Dang! M7 X is a 2020 ☹️
